admin/navigation.json

```diff
@@ -1,5 +1,6 @@
 {
-  "section": "other",
+  "LMProductId": "6",
+  "section": "transactions",
```

Comment:
Please update the docs (step 3) to explain where to find the CyberSource admin link

Like: "In VTEX Admin, select `Transactions > CyberSource` and..."

Reply:
updated